### DISCUSSION TITLE
Meta: cnschema 志愿者

### DISCUSSION THREAD

**Discussion by lidingpku** [cnschema/cnschema#64] [General]
希望参与到cnSchema志愿工作可以回复此issue，说明您的兴趣，能贡献的能力，给出一个例子说明cnschema如怎样能帮助您的实际工作。

**Comment by du00cs** [comment]
我个人主要处理与人物相关的，包括人物属性和相关的概念（比如学校、出生地、作品）。个人比较关心的（也是正在做的）是将cnschema在应用的时候转换到RDFS/OWL的框架下，然后借助自动代码生成来减少使用复杂度，最后导入到RDF数据库，这一套流程性的东西。不清楚我这个路线是否正确，还请帮忙判断判断。

转换时候的步骤：
1. 为cnschema的json增加必要的部件，完备定义（目前有所缺失，直接在`cns.py`中添加代码我还没太弄懂，目前是在结果上直接增加的）
2. 改写项目 [schemaorg-transformer](https://github.com/vcharpenay/schemaorg-transform) 支持对cnschema转换
3. 利用Apache Jena的jena.shemagen自动生成代码（这里可以考虑按照概念进行拆分，便于使用）
4. 利用自动生成的代码重新生成数据

**Comment by lidingpku** [comment]
欢迎， https://github.com/schemaorg/schemaorg/blob/master/data/releases/3.2/schema.rdf 已经有RDFS／OWL的定义了。 cnschema这边会尽快发布类似的定义说明。 JSON-LD是中间态。但是，要注意编程逻辑，我们倾向基于interface的composition 方法，而不是限于类的继承
 https://github.com/cnschema/cnschema/issues/35

**Comment by sunerye** [comment]
我可以参与项目管理

**Comment by GithubDaizhen** [comment]
一直主导公司的知识图谱schema建设方面的工作。编写过owl格式的ontology生成工具，搭建过webprotege的环境，使用gojs开发过schema可视化有向无环图diagram, 规划设计过近300个概念,5级schema的方案。
希望有机会加入cnSchema的建设和维护工作的队伍中来。
